ras_backstage/resources/collection_exercise/single_collection_exercise.py

```diff
@@ -43,6 +47,7 @@ def get(short_name, period):
             "collection_exercise": full_exercise,
             "events": exercise_events,
             "collection_instruments": collection_instruments,
+            "eq_ci_selectors": eq_ci_selectors,
```

Comment:
Are we missing a test to check the eq_ci_selectors?

Reply:
I've just added an assert in the test_single_collection_exercise to check that eq_ci_selectors are being passed back in the response.